packages/fondue/src/components/RichTextEditor/serializer/markdown/__tests__/MarkdownToSlate.spec.ts

```diff
@@ -61,8 +61,14 @@ describe('Markdown to slate Transformer', () => {
     });
 
     it('should transform paragraph', () => {
```

Comment:
```typescript
 const testCases = [
        { markdown: codeBlockMarkdown[0], expectedTree: codeBlockTree[0] },
        { markdown: codeBlockMarkdown[1], expectedTree: codeBlockTree[1] },
        { markdown: codeBlockMarkdown[2], expectedTree: codeBlockTree[2] },
    ];
    
    testCases.forEach(({ markdown, expectedTree }) => {
        it(`should transform code block`, () => {
            const result = transformer.process(markdown);
            expect(result).to.deep.equal(expectedTree);
        });
    });
``` 
How about that?

Would do it like that also for the other tests that are adjusted, as its odd to have multiple expects, or if there are different things you test then split it to multiple tests with proper naming

Reply:
I think you can not do that in cypress, just vi

Reply:
Updated

Reply:
Nice, thanks 👍